### ISSUE TITLE
Video Description

### ISSUE THREAD

**Issue by UncleKaleKale** [UncleKaleKale/susie-creative#9]
Need to give videos a description on [videos page](https://github.com/UncleKaleKale/susie-creative/commit/a0fbf4d9d467c80d55314a9bdcae6b07d6898a0c).

**Comment by coltenkrauter** [comment]
Can you share some more details?

Are you looking to add some text beneath each video?

**Comment by UncleKaleKale** [comment]
Yes maybe sorta like a YouTube video. A white background behind each video
with text on it might be nice, with the normal background behind all that
like it is. Mostly just adding white box like extra padding around video or
just at bottom with text at bottom. Maybe still with background picture
visible in between the videos like it is
